tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -228,6 +236,15 @@ Given('the administrator has excluded group {string} from sharing', async functi
   }
 })
 
+Given('user {string} has created a share with following settings',
+  function (sharer, dataTable) {
+    const setting = dataTable.rowsHash()
+    return shareFileFolder(
```

Comment:
This takes a whole lot of arguments, would be good to think of some ways to cope with this.

Reply:
Please checkout what I've done!